source.go

```diff
@@ -6,24 +6,30 @@ import (
 	"strings"
 )
 
-// SourceDirSubdir takes a source and returns a tuple of the URL without
-// the subdir and the URL with the subdir.
-func SourceDirSubdir(src string) (string, string) {
+// SourceDirSubdir takes a source and returns the URL without
+// the subdir and the subdir.
+func SourceDirSubdir(src string) (url string, subdir string) {
```

Comment:
The prameterized return values here should be removed.

Just looking at the code there does not seem to be any changes introduced that would require a naked return. I'm guessing these were added to save on possible variable declarations - if that's the case, they should be just added as variables either with `var` or initialization operators.

Reply:
Yeah, don't add named returns just because you don't know what they are. This ultimately creates addressable variables for them the function. If it's a documentation issue, add documentation.

Reply:
Gotcha, will change this 🙂